include/hx/CFFINekoLoader.h

```diff
@@ -230,25 +230,26 @@ int api_val_strlen(neko_value  arg1)
 	return 0;
 }
 void api_buffer_set_size(neko_buffer inBuffer,int inLen) { 
-   NEKO_NOT_IMPLEMENTED("api_buffer_set_size");
+  // NEKO_NOT_IMPLEMENTED("api_buffer_set_size");

```

Comment:
What is this? I suppose these Neko changes are not actually related to the GC improvements?

Reply:
i was building my project with "openfl build windows" and it was giving the error api_buffer_set_size NOT IMPLEMENTED while trying to copy the ico files into the executable. I've reverted some changes in the CFFINekoLoader to let it work.

Reply:
Still I don't think it should be part of this PR, it really sounds like an unrelated issue.

Reply:
Indeed it's unrelated